src/System.Runtime.Serialization.Xml/System.Runtime.Serialization.Xml.sln

```diff
@@ -2,6 +2,11 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
 VisualStudioVersion = 14.0.25420.1
 MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "System.Runtime.Serialization.Xml.CN.Tests", "tests\Canonicalization\System.Runtime.Serialization.Xml.CN.Tests.csproj", "{D3AD82FC-051D-4121-8E48-B13CE8024776}"
```

Comment:
The new project's GUID is newly created, it is "{D3AD82FC-051D-4121-8E48-B13CE8024776}", I then updated the .sln file manually by referring to the other test project config in this solution. I have few knowledge of the GUID things here, by "create new GUID for the new project", could you plz let me know which part of the **following GUID** should be updated?

Project("{**FAE04EC0-301F-11D3-BF4B-00C04F79EFBC**}") = "System.Runtime.Serialization.Xml.CN.Tests", "tests\Canonicalization\System.Runtime.Serialization.Xml.CN.Tests.csproj", "{**D3AD82FC-051D-4121-8E48-B13CE8024776**}"
	ProjectSection(ProjectDependencies) = postProject
		**{9D747A18-C8FD-4D7A-8913-4ED7911683B4} = {9D747A18-C8FD-4D7A-8913-4ED7911683B4}**
	EndProjectSection
EndProject